packages/my-near-wallet/src/lib/my-near-wallet.ts

```diff
@@ -209,7 +206,7 @@ export function setupMyNearWallet({
       id: "my-near-wallet",
       type: "browser",
       metadata: {
-        name: "My NEAR Wallet",
+        name: "MyNearWallet",
```

Comment:
Why MyNearWallet? Is the name of the metadata used to display on the page?

Reply:
Yeah. The formal brand name is `MyNearWallet`